alcs-api/src/application/application-paused.service.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { createMock, DeepMocked } from '@golevelup/nestjs-testing';
```

Comment:
maybe rename file to application-time-tracking.service.spec.ts?

Reply:
Good catch, definitely should be fixed!

Reply:
Done!